website/pages/docs/auth/jwt_oidc_providers.mdx

```diff
@@ -70,6 +70,36 @@ vault write auth/oidc/role/demo user_claim="email" \
     policies=default
 ```
 
+### Azure-specific handling configuration
+
+If a user is a member of more than 200 groups (directly or indirectly), extra configuration is required so that vault can fetch the groups properly.
+
+- Set `"provider_config"` to azure (see below)
+- Grant the AAD application `Directory.Read.All` permission under "Azure Active Directory Graph" API Permissions (legacy API).
+- Add `"profile"` to `oidc_scopes` so the user's id comes back on the jwt token.
+
+```
+vault write auth/oidc/config -<<"EOH"
+{
+   "oidc_client_id": "your_client_id",
+   "oidc_client_secret": "your_client_secret",
+   "default_role": “your_default_role”,
+   "oidc_discovery_url": "https://login.microsoftonline.com/tenant_id/v2.0”,
+   "provider_config": {
+      "provider": "azure"
+   }
+}
```

Comment:
Smart quotes cause copy/paste problems.  Removed them:

```suggestion
vault write auth/oidc/config -<<"EOH"
{
   "oidc_client_id": "your_client_id",
   "oidc_client_secret": "your_client_secret",
   "default_role": "your_default_role",
   "oidc_discovery_url": "https://login.microsoftonline.com/tenant_id/v2.0",
   "provider_config": {
      "provider": "azure"
   }
}
```

Reply:
Thanks, looks like I got the quotes from the examples above. Fixing those too.